src/Codeception/Attribute/Given.php

```diff
@@ -1,10 +1,15 @@
 <?php
 
+declare(strict_types=1);
+
 namespace Codeception\Attribute;
 
 use Attribute;
 
 #[Attribute(Attribute::TARGET_METHOD | Attribute::IS_REPEATABLE)]
 final class Given
 {
+    public function __construct(string $description)
```

Comment:
Also repeatable, but I haven't tested this attribute. Might be one to change to `string ...$descriptions`.

Reply:
Multiple descriptions per Given don't seem to be a thing.

Reply:
Yep, a misunderstanding by me re: the `IS_REPEATABLE` flag for an attribute. Case closed 🔨